tests/spdx3/fixtures.py

```diff
@@ -298,6 +309,42 @@ def namespace_map_fixture(
     "relationship_type": RelationshipType.FIXED_IN,
 }
 
+ARTIFACT_DICT = {
+    "originated_by": ["https://spdx.test/tools-python/originatedBy"],
+    "supplied_by": ["https://spdx.test/tools-python/suppliedBy"],
+    "built_time": datetime(2004, 1, 1),
+    "release_time": datetime(2005, 1, 1),
+    "valid_until_time": datetime(2006, 1, 1),
+    "standard": ["https://spdx.test/tools-python/standard"],
+}
+
+SOFTWARE_ARTIFACT_DICT = {
+    "content_identifier": "contentIidentifier",
```

Comment:
`content_identifier` must be a URI, see [here](https://github.com/spdx/spdx-3-model/blob/main/model/Software/Properties/contentIdentifier.md)
```suggestion
    "content_identifier": ""https://spdx.test/tools-python/contentIdentifier"",
```